packages/sync/src/ProxyUpdate.ts

```diff
@@ -0,0 +1,5 @@
+import { MutationUpdaterFn } from "apollo-client";
+
+type ProxyUpdate = (mutationName: string) => MutationUpdaterFn;
```

Comment:
I see aproach from: https://medium.com/twostoryrobot/a-recipe-for-offline-support-in-react-apollo-571ad7e6f7f4